drake_cmake_installed/src/simple_pcl_test/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+find_package(PCL REQUIRED)
```

Comment:
Can you add a comment here to articulate that this must be called *after* a find_package(drake) has been issued.